pkg/datapath/loader/base.go

```diff
@@ -363,6 +364,15 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		args[initArgTunnelMode] = option.TunnelVXLAN
 	}
 
+	switch args[initArgTunnelMode] {
+	case option.TunnelVXLAN:
+		args[initArgTunnelPort] = fmt.Sprintf("%d", option.TunnelPortDefaultVXLAN)
+	case option.TunnelGeneve:
+		args[initArgTunnelPort] = fmt.Sprintf("%d", option.TunnelPortDefaultGeneve)
+	default:
+		args[initArgTunnelPort] = "<nil>"
```

Comment:
@jibi did you see this? I'm just following the general patter of how undefined arguments are handled by this function.